edx_name_affirmation/toggles.py

```diff
@@ -0,0 +1,19 @@
+"""
+Toggles for edx-name-affirmation app
+"""
+
+from edx_toggles.toggles import WaffleFlag
+
+# .. toggle_name: name_affirmation.verified_name
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Enable the verified name feature
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2021-06-25
```

Comment:
It's not a big deal, but it might be a good idea to update this before you merge.